bin/find-never-defined-css-variables/find-never-defined-css-variables.ts

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env tsx
+/* eslint-disable no-console */
+import { readFileSync } from "fs";
+
+import { glob } from "glob";
+
+/**
+ * This script finds css variables that are used but *never* defined in our codebase.
+ * Its goal isn't to find usages of variables in contexts where that variable is not defined,
+ * as that would be a lot harder.
+ * Its goal is to prevent typos and wrong usages created by conflicts and backports.
+ * The most common thing this should catch and prevent is backporting a fix or feature that uses a css variable
+ * that doesn't exist in the release branch.
+ */
+
+const shouldWhiteList = (_variable: string) => {
+  // Use this for whitelisting some variables if we know they're defined by a 3rd party, such as mantine
+  // Note: for be sure to not backport changes to this function that would whitelist variables not on the release branch
```

Comment:
```suggestion
  // Note: Be sure not to backport changes to this function that would whitelist variables not on the release branch
```

Or maybe you should change the function name to like `shouldWhiteListThirdPartyCssVariables` or something. I assume all our own variables shouldn't be whitelisted correct?

Reply:
In the mantine branch we ended up parsing the mantine css file so we don't have to manually whitelist them.
We still have a (hopefully temporary) whitelist to not block merging the mantine branch